### DISCUSSION TITLE
Exploding Coilgun Ammunition Overpowered?

### DISCUSSION THREAD

**Discussion by MarkCringan** [FakeFishGames/Barotrauma#13464] [Polls]
With exploding Coilgun Ammunition you can turn a 250 Deep Tissue Injury (DTI) block of C-4 into 5,500 (DTI) worth of coilgun damage.

Is this balanced? If not, how would you want it reworked? Do you think it should be removed?

**Comment by bookslayer10** [comment]
While I do think it's overly powerful, it's also really fun to use, so changing it isn't a high priority IMO.

I think moderately nerfing the single target damage will be good enough, as well as reducing the structural damage. Explosive ammo is really good DPS.

**Comment by DominO570** [comment]
I think it's not normal that you can make boxes of 5 on the initial outpost, I don't like that situation and it makes them the best for coilgun. It would be worth bringing back the old recipe using c-4+.

**Comment by LePiromano** [comment]
there's already a commit in the github mentioning a nerf to the exploding coilgun ammo but it's not present in the current stable or unstable builds yet, but yeah exploding ammo is broken op currently
